appinventor/appengine/src/com/google/appinventor/client/editor/simple/components/MockTableLayout.java

```diff
@@ -436,4 +450,29 @@ void dispose() {
       DOM.removeChild(container.getRootPanel().getElement(), dropTargetArea);
     }
   }
+
+  private boolean isInside(Widget w, int absX, int absY) {
+    int wx = w.getAbsoluteLeft();
+    int wy = w.getAbsoluteTop();
+    int ww = w.getOffsetWidth();
+    int wh = w.getOffsetHeight();
+    return (wx <= absX) && (absX < wx + ww) && (wy <= absY) && (absY < wy + wh);
+  }
+
+  // Sets a border around cell when mouse is clicked on a cell in table
+
+  public void setCell(Widget table, Widget sender, int x, int y) {
```

Comment:
Maybe a slightly more descriptive name for this function, e.g., `setTargetCell`, `setActiveCell` or `setSelectedCell`.